students/JaeKim/lesson07/assignment/linear.py

```diff
@@ -0,0 +1,105 @@
+import os
+import csv
+from pymongo import MongoClient
+
+
+class MongoDBConnection():
+    """MongoDB Connection"""
+
+    def __init__(self, host='127.0.0.1', port=27017):
+        self.host = host
+        self.port = port
+        self.connection = None
+
+    def __enter__(self):
+        self.connection = MongoClient(self.host, self.port)
+        return self
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        self.connection.close()
+
+
+def cleanup():
+    mongo = MongoDBConnection()
+
+    with mongo:
+        database = mongo.connection.media
+
+        [i.drop() for i in [database.products, database.customers, database.rentals]]
+
+
+def import_data(directory_name, product_file, customer_file, rentals_file):
+    mongo = MongoDBConnection()
+
+    with mongo:
+        database = mongo.connection.media
+        products_csv = os.path.join(directory_name, product_file)
+        customer_csv = os.path.join(directory_name, customer_file)
+        rentals_csv = os.path.join(directory_name, rentals_file)
+
+        products_data = csv.reader(open(products_csv, encoding='utf-8-sig'))
+        customers_data = csv.reader(open(customer_csv, encoding='utf-8-sig'))
+        rentals_data = csv.reader(open(rentals_csv, encoding='utf-8-sig'))
+
+        collection_products, collection_customers, collection_rentals = database['products'], database['customers'], database['rentals']
+        data = [products_data, customers_data, rentals_data]
+
+        [_insert_data(x, y) for x, y in zip([collection_products, collection_customers, collection_rentals], data)]
```

Comment:
Again, here you aren't using the list for anything, so don't bother creating it. 

Aren't you suppose to be collecting errors and number of entries and returning those?